File: GTTS/INL.TrainingService.Models/SaveTrainingEventCourseDefinition_Param.cs

namespace INL.TrainingService.Models
{
    public class SaveTrainingEventCourseDefinition_Param : ISaveTrainingEventCourseDefinition_Param
    {
        public long TrainingEventID { get; set; }
        public int? CourseDefinitionID { get; set; }
        public byte? TestsWeighting { get; set; }
        public byte? PerformanceWeighting { get; set; }
        public byte? ProductsWeighting { get; set; }
        public byte? MinimumAttendance { get; set; }
        public byte? MinimumFinalGrade { get; set; }
        public bool IsActive { get; set; }
        public int ModifiedByAppUserID { get; set; }
    }
}

File: GTTS/GTTS.Web/ClientApp/src/app/models/INL.TrainingService.Models/save-training-event-course-definition_param.ts


import { ISaveTrainingEventCourseDefinition_Param } from './isave-training-event-course-definition_param';

export class SaveTrainingEventCourseDefinition_Param implements ISaveTrainingEventCourseDefinition_Param {
  
	public TrainingEventID: number = 0;
	public CourseDefinitionID?: number;
	public TestsWeighting?: number;
	public PerformanceWeighting?: number;
	public ProductsWeighting?: number;
	public MinimumAttendance?: number;
	public MinimumFinalGrade?: number;
	public IsActive: boolean = false;
  
}






